.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+env:
+  RUSTFLAGS: -Dwarnings
+  RUST_BACKTRACE: 1
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+      - name: Install Rust (rustup)
+        run: rustup update stable --no-self-update && rustup default stable
+      - name: Install Clang-12
+        uses: KyleMayes/install-llvm-action@v1
+        with:
+          version: "12.0.0"
+          directory: ${{ runner.temp }}/llvm
+      - name: Add clang++-12 link
+        working-directory: ${{ runner.temp }}/llvm/bin
+        run: ln -s clang clang++-12
+      - name: Check formatting
+        run: cargo fmt --all -- --check
+      - name: Build rdp-rs
+        run: cargo build --verbose --all --all-features
```

Comment:
I wouldn't do `--all-features` - we don't care about `mstsc-rs` at all (it might not even build today).

Reply:
Well, with my changes, it builds now. Up to you. :-)

Reply:
I don't much care either way. Like I said, this repo is going to the graveyard soon.